apps/mobile/src/queries/transaction/use-check-utxos.ts

```diff
@@ -0,0 +1,139 @@
+import { useCallback, useState } from 'react';
+
+import { TransactionInput } from '@scure/btc-signer/psbt';
+import { bytesToHex } from '@stacks/common';
+
+import { BitcoinClient, getNumberOfInscriptionOnUtxoUsingOrdinalsCom } from '@leather.io/query';
+import { isUndefined } from '@leather.io/utils';
+
+import { useBitcoinClient } from '../clients/bitcoin-client';
+import { useCurrentNetworkState, useIsLeatherTestingEnv } from '../leather-query-provider';
+
+class PreventTransactionError extends Error {
+  constructor(message: string) {
+    super(message);
+    // eslint-disable-next-line lingui/no-unlocalized-strings
```

Comment:
A side note that this `lingui/no-unlocalized-strings` rule is annoying sometimes 

Maybe this can instead use `BitcoinError` which I moved to `packages/bitcoin/`?